sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetRenderer.kt

```diff
@@ -254,35 +253,43 @@ internal class BitmapWidgetRenderer(
     this.bitmap = bitmap
     this.halfBitmapWidth = bitmap.width / 2f
     this.halfBitmapHeight = bitmap.height / 2f
-    setTranslation(this.translationX, this.translationY)
+    updatePosition(widgetPosition)
     updateVertexBuffer()
     updateMatrix = true
     needRender = true
   }
 
   override fun setRotation(angleDegrees: Float) {
+    rotation = angleDegrees
     Matrix.setIdentityM(rotationMatrix, 0)
     Matrix.setRotateM(rotationMatrix, 0, angleDegrees, 0f, 0f, 1f)
     updateMatrix = true
     needRender = true
   }
 
-  override fun setTranslation(translationX: Float, translationY: Float) {
-    this.translationX = translationX
-    this.translationY = translationY
+  override fun getRotation(): Float {
+    return rotation
+  }
+
+  override fun updatePosition(widgetPosition: WidgetPosition) {
+    this.widgetPosition = widgetPosition
```

Comment:
Should this line (275) be separated from the matrix updates? We seem to be calling this `updatePosition` internally in this class with `this.widgetPosition` as parameter.

Something like:
```Kotlin
override fun updatePosition(widgetPosition: WidgetPosition) {
    this.widgetPosition = widgetPosition
    updateTranslateMatrix()
  }

  private fun updateTranslateMatrix() {
    Matrix.setIdentityM(translateMatrix, 0)
    Matrix.translateM(
      translateMatrix,
      0,
      leftX(),
      topY(),
      0f
      )

    updateMatrix = true
    needRender = true
  }
```
and then `updateBitmap` and `onSurfaceChanged` only need to call to `updateTranslateMatrix`.
What do you think?

Reply:
Sounds good, updated.